### PR TITLE
feat(encoder): add tfkeras mobilenet encoder

### PR DESCRIPTION
**Pull request by kashyapvad**

### CODE REVIEW

File: hub/executors/encoders/image/tfkeras-mobilenet_v2/manifest.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name: tfkeras mobilenet_v2
+description: an image with the encoder of mobilenet.v2 from tfkeras
+author: Jina AI Dev-Team (dev-team@jina.ai)
+url: https://jina.ai
+documentation: https://github.com/jina-ai/jina-hub
+version: 0.0.0
+vendor: Jina AI Limited
+license: apache-2.0
```
Comment (nan-wang) [inline hub/executors/encoders/image/tfkeras-mobilenet_v2/manifest.yml:8]: missing `update` key. Would you please have a look at [`update`](https://github.com/jina-ai/jina-hub#schema-of-manifestyml) in the documentation?

File: hub/executors/encoders/image/tfkeras-mobilenet_v2/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+FROM jinaai/jina:devel
+
+RUN pip install tensorflow
```
Comment (nan-wang) [inline hub/executors/encoders/image/tfkeras-mobilenet_v2/Dockerfile:3]: Following changes will speed up the pulling speed if user has the `tensorflow` image, which is more likely than having the `jina` image. ;)
```
FROM tensorflow/tensorflow:latest
RUN pip install jina[devel]
```

**Comment by kashyapvad** [comment]
Noted!

File: hub/executors/encoders/image/tfkeras-mobilenet_v2/manifest.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+name: tfkeras mobilenet_v2
+description: an image with the encoder of mobilenet.v2 from tfkeras
+author: Jina AI Dev-Team (dev-team@jina.ai)
```
Comment (nan-wang) [inline hub/executors/encoders/image/tfkeras-mobilenet_v2/manifest.yml:3]: @kashyapvad Last but not the least, it is our great honor to have your name listed here. ;)

```suggestion
author: Jina AI Dev-Team (dev-team@jina.ai)
```

Comment (kashyapvad) [reply hub/executors/encoders/image/tfkeras-mobilenet_v2/manifest.yml:3]: The honor's mine :)

**Pull request closed without merging**